doc/source/index.rst

```diff
@@ -1,116 +1,106 @@
-SciPy
-=====
+.. module:: scipy
 
-:Release: |release|
-:Date: |today|
+*******************
+SciPy documentation
+*******************
 
-SciPy (pronounced "Sigh Pie") is open-source software for mathematics,
-science, and engineering.
+**Date**: |today| **Version**: |version|
 
-.. toctree::
-   :maxdepth: 1
+**Download documentation**: `PDF Version <scipy.pdf>`__ | `Zipped HTML <scipy.zip>`__
 
-   install_upgrade
-   api
-   release
+**Useful links**:
+`Binary Installers <https://pypi.org/project/scipy>`__ |
+`Source Repository <https://github.com/scipy/scipy>`__ |
+`Issues & Ideas <https://github.com/scipy/scipy/issues>`__ |
+`Q&A Support <https://stackoverflow.com/questions/tagged/scipy>`__ |
+`Mailing List <https://mail.python.org/pipermail/scipy-user>`__
 
-Tutorial
---------
+:mod:`scipy` (pronounced "Sigh Pie") is open-source software for mathematics,
+science, and engineering.
 
-Tutorials with worked examples and background information for most SciPy
-submodules.
+.. raw:: html
 
-.. toctree::
-   :maxdepth: 2
+    <div class="container">
+        <div class="row">
+            <div class="col-lg-6 col-md-6 col-sm-6 col-xs-12 d-flex">
+                <div class="card text-center intro-card shadow">
+                <img src="_static/index_getting_started.svg" class="card-img-top" alt="getting started with SciPy action icon" height="52">
```

Comment:
I like the icons! Can we add a little padding between the top of them and the top of the box they're in?

![image](https://user-images.githubusercontent.com/6570539/112003463-2995fb80-8ade-11eb-91b1-41ac933e3dbc.png)

Reply:
Sure I guess! I just copied over the code from Pandas for now. I didn't play around too much as I was not sure if we where going to use all these. I am not sure why on Pandas site they have the padding and not here (same in the `Getting started`).

Reply:
Speaking of icons, we could also add NumFOCUS and Tidelift along with GitHub's icon.

Reply:
I think we want those NumFOCUS and Tidelift (and CZI, and potentially other supporters in the future) icons on the front page (i.e. scipy.org, which we should redesign too), and not inside the docs for one version of SciPy.